templates/compat/themes/betheme/fc/checkout-page-template/checkout/page-checkout.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * The checkout template file.
+ *
+ * This template can be overridden by copying it to yourtheme/woocommerce/checkout/page-checkout.php.
+ *
+ * HOWEVER, on occasion WooCommerce will need to update template files and you
+ * (the theme developer) will need to copy the new files to your theme to
+ * maintain compatibility. We try to do this as little as possible, but it does
+ * happen. When this occurs the version of the template file will be bumped and
+ * the readme will list any important changes.
+ *
+ * @see     https://docs.woocommerce.com/document/template-structure/
+ * @package fluid-checkout
+ * @version 3.0.4
+ * @fc-version 3.1.4
```

Comment:
Do not add `@fc-version` for files originally created by Fluid Checkout. In these cases you don't have to add or change the version number.

You can determine if the file is original to Fluid Checkout when the `@package` is `fluid-checkout` or the folder name of any of our other plugins.